packages/odyssey-react-mui/src/labs/Switch.tsx

```diff
@@ -127,7 +141,9 @@ const SwitchThumb = styled("span", {
     borderRadius: odysseyDesignTokens.BorderRadiusRound,
     backgroundColor: isDisabled
       ? odysseyDesignTokens.HueNeutral50
-      : odysseyDesignTokens.HueNeutralWhite,
+      : isReadOnly
```

Comment:
I guess if we are moving away from the ternary approach in the styles, we might as well update these as well